Description: silent remove excessive verbosity at startup about conf.d
 directory parsing.
Author: Francesco Paolo Lovergine <frankie@debian.org>
Forwarded: not needed

Index: proftpd-dfsg/src/dirtree.c
===================================================================
--- proftpd-dfsg.orig/src/dirtree.c	2013-06-24 17:56:32.000000000 +0200
+++ proftpd-dfsg/src/dirtree.c	2013-06-24 17:56:32.000000000 +0200
@@ -3262,9 +3262,6 @@
       }
     }
 
-    pr_log_pri(PR_LOG_INFO, "processing configuration directory '%s'",
-      dup_path);
-
     dirh = pr_fsio_opendir(dup_path);
     if (!dirh) {
       pr_log_pri(PR_LOG_ERR,
